habitat/core/gym_env_episode_count_wrapper.py

```diff
@@ -37,6 +37,13 @@ def current_episode(self) -> Episode:
                 start_rotation=[],
             )
 
+    @property
+    def original_action_space(self) -> spaces.space:
```

Comment:
Hmm this is not ideal. This property will need to be exposed on every gym wrapper. Is there a way we can automatically have this property for every wrapper?

Reply:
> This property will need to be exposed on every gym wrapper

gym.Wrapper forwards property calls to the underlying environment: https://github.com/openai/gym/blob/master/gym/core.py#L253

This means that even if we create more wrappers, the property original_action_space will always be here.

Was that your concern or did I misunderstand ?

Reply:
Ah got it. Yes this addresses my concern.